lib/isl/README.md

```diff
@@ -1,4 +1,7 @@
-## PartiQL ISL Kotlin
+## PartiQL ISL Kotlin (Testing Utility)
+
+**NOTE**: PartiQL ISL is no longer published to Maven. The generated classes should only be used for
```

Comment:
Are you able to confirm that the generated sources do not make it into the `partiql-lang-kotlin` JAR?

Reply:
Yes.
```
unzip ~/.m2/repository/org/partiql/partiql-lang-kotlin/0.11.1-SNAPSHOT/partiql-lang-kotlin-0.11.1-SNAPSHOT.jar -d ~/Downloads/0.11.1/

tree ~/Downloads/0.11.1 | grep ionschema
```

Nothing is shown. Grepping `isl` only shows the resource files as well as the `partiqlisl` package.